cybereason_connector.py

```diff
@@ -945,6 +948,54 @@ def _get_machine_name_by_machine_ip(self, machine_ip, action_result):
 
         return RetVal(action_result.set_status(phantom.APP_SUCCESS), machine_names)
 
+    def on_poll(self, param):
+        self.save_progress("Entered the function")
```

Comment:
"Entered the function" is pretty generic and you're doing it for several - you should probably include the function name.

Reply:
done